### ISSUE TITLE
no module name tensorflow_probability 

### ISSUE THREAD

**Issue by devbhanu** [deepmind/graph_nets#3]
import graph_nets as gp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bhanuteja/anaconda3/lib/python3.6/site-packages/graph_nets/__init__.py", line 21, in <module>
    from graph_nets import blocks
  File "/home/desktopname/anaconda3/lib/python3.6/site-packages/graph_nets/blocks.py", line 38, in <module>
    import sonnet as snt
  File "/home/desktopname /anaconda3/lib/python3.6/site-packages/sonnet/__init__.py", line 42, in <module>
    from sonnet.python import custom_getters
  File "/home/desktopname/anaconda3/lib/python3.6/site-packages/sonnet/python/custom_getters/__init__.py", line 21, in <module>
    from sonnet.python.custom_getters import bayes_by_backprop
  File "/home/desktopname/anaconda3/lib/python3.6/site-packages/sonnet/python/custom_getters/bayes_by_backprop.py", line 94, in <module>
    import tensorflow_probability as tfp

**Comment by huawei-tomas** [comment]
Go install it.
https://github.com/tensorflow/probability

**Comment by alvarosg** [comment]
See [this issue](https://github.com/deepmind/graph_nets/issues/2).

The current fix is: `pip install -U dm-sonnet==1.23`

**Comment by jhamrick** [comment]
For now, we have released a version which pins the version of sonnet to 1.23. Let's keep this issue open though so we remember to unpin it after the problem is fixed upstream.

**Comment by IMBurbank** [comment]
I haven't checked Colab, but Graph Nets seems to work locally with a fresh install. 

I just checked the Docker images I proposed in PR #4 in light of this bug label. Those images only pull from Tensorflow base-images and follow your current install instructions.

```dockerfile
FROM tensorflow/tensorflow

RUN pip install \
        graph_nets \
        tensorflow_probability \
...
```

`dm-sonnet` isn't explicitly installed, and all of the images run the tests and demos successfully.

For example, using the Graph Nets images I'm currently [hosting](https://hub.docker.com/r/imburbank/graph_nets/)

```bash
docker run --rm -u $(id -u):$(id -g) -p 8888:8888 -it imburbank/graph_nets:latest-demos
```

I just ran the shortest_path.

![shortest-path-screenshot](https://user-images.githubusercontent.com/21133719/47321777-12a00600-d613-11e8-9f78-b7c929322de7.png)
